Documentation/ignition.md

```diff
@@ -44,7 +46,7 @@ ignition/network.tmpl:
             {{end}}
     {{end}}
 
-Response from `/ignition?mac=address` for a particular machine.
+Below is a response that you would get from making a GET request to `/ignition?mac=address` for a particular machine. As you can see, the file response is in JSON, not YAML, as Ignition only supports JSON. You can also see that the templated values such as `{{.networkd_name}}` has been filled in.
```

Comment:
perhaps "as Ignition is a JSON format". To make it more clear it is an intentional choice on Ignition's part.